test/test.js

```diff
@@ -249,3 +250,29 @@ test('multiple hook.unhook()', function (t) {
   hook2.unhook()
   t.end()
 })
+
+if (semver.lt(process.version, '12.0.0')) {
```

Comment:
The reason for the if-statement is that in Node.js 12.0.0+ there are no more core modules that contain slashes.